### PR TITLE
Current position as JSON

### PR DESCRIPTION
**Pull request by sistemico**
After this change current position value could be any JSON value (number, string, boolean or object) without break the current API

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -46,20 +46,28 @@ exports.Exporter = class {
   async getLastPosition() {
     if (await zookeeperClient.existsAsync(this.zookeeperPositionNode)) {
       const previousBlockNumber = await zookeeperClient.getDataAsync(this.zookeeperPositionNode)
-      return previousBlockNumber.data.readUInt32BE(0)
-    } else {
-      return null
+
+      if (previousBlockNumber && Buffer.isBuffer(previousBlockNumber.data)) {
+        try {
+          return JSON.parse(previousBlockNumber.data.toString('utf8'))
```
Comment (valo) [inline index.js]: I think that won’t be backward compatible with the old API that works with numbers. Can we make it smart and continue supporting the old format?

Comment (sistemico) [reply index.js]: Do you have an idea how to "detect" if a value is in the old format?

Comment (sistemico) [reply index.js]: Added a format header to detect new values

**Comment by valo** [comment]
Thanks for contributing <3

**Pull request merged**